stream_alert_cli/terraform/generate.py

```diff
@@ -42,6 +42,7 @@
 )
 from stream_alert_cli.terraform.monitoring import generate_monitoring
 from stream_alert_cli.terraform.streamalert import generate_stream_alert
+from stream_alert_cli.terraform.rule_promotion import generate_rule_promotion
```

Comment:
Misalphetization: `r < s`. I don't know why that sticks out to me so much, but it does

Reply:
fixed! thanks ;)